openvpn/terraform/gcp/main.tf

```diff
@@ -0,0 +1,11 @@
+resource "kubernetes_namespace" "openvpn" {
```

Comment:
Since gcp, aws, azure, etc really have no cloud interaction here, should we perhaps merge these into a single terraform module supporting all providers?

Reply:
Agreed